### ISSUE TITLE
Pi3

### ISSUE THREAD

**Issue by Dragonball1472** [RemovedMoney326/Hubble-Pi#5]
Funktioniert das skript auch mit einem Raspberry Pi 3?

**Comment by Mraanderson** [comment]
Yes, the document states (2.1 point 1) any Raspberry Pi will work. I currently use a Raspberry Pi Zero W with an alternative, yet similar setup. 

Good luck.

**Comment by kristianschneider** [comment]
> Yes, the document states (2.1 point 1) any Raspberry Pi will work. I currently use a Raspberry Pi Zero W with an alternative, yet similar setup.
> 
> Good luck.

How does the Zero W perform? For example something like plate solving or polar alignment, is that doable?
